src/lib/actionrunner.js

```diff
@@ -287,6 +288,11 @@ class OpenwhiskActionRunner {
     _getImage() {
         // blackbox -> image is specified directly
         if (this.action.exec.image) {
+            if(process.env.ASSET_COMPUTE_DOCKER_IMAGE_PREFIX 
```

Comment:
I would name this `ADOBE_RUNTIME_DOCKER_REGISTRY` (it really is the registry not the prefix, the prefix would be `nui/`) and use this name consistently everywhere including our CI: https://git.corp.adobe.com/nui/api-process/pull/81#discussion_r3122910

cc @pheenomenon